_posts/2022-5-18-introducing-accelerated-pytorch-training-on-mac.md

```diff
@@ -25,6 +25,11 @@ In the graphs below, you can see the performance speedup from accelerated GPU tr
   <img src="/assets/images/METAPT-002-BarGraph-02.gif" width="100%">
 </p>
 
+<p align = "center">
```

Comment:
Should this be part of the same paragraph?

Reply:
Require more information. This should be the image with caption and additional space after caption to the header.

<p align="center">
  <img src="/assets/images/METAPT-002-BarGraph-02.gif" width="100%">
</p>

<p align = "center">
Accelerated GPU training and evaluation speedups over CPU-only (times faster)
</p>

When looking at the preview, this displays correct.